apps/test/unit/templates/SchoolAutocompleteDropdownTest.js

```diff
@@ -1,78 +1,90 @@
 import React from 'react';
 import {mount} from 'enzyme';
-import {expect} from '../../util/deprecatedChai';
+import {assert, expect} from 'chai';
```

Comment:
What's the difference between `reconfiguredChai` and importing `chai` directly?

Reply:
`reconfiguredChai.js` imports `chai`, configures it to use a number of extensions and some custom settings, then re-exports it.

https://github.com/code-dot-org/code-dot-org/blob/b09c32514d3b864e5739a222de233c7b47684705/apps/test/util/reconfiguredChai.js#L8-L13

By importing `chai` directly, I'm saying that this test file doesn't depend on any of those extensions (although they will still be present if this file is run in a bundle after another test that imports `reconfiguredChai`).  I've been moving this direction because we've had issues with chai extensions in the past (hence [deprecatedChai.js](https://github.com/code-dot-org/code-dot-org/blob/b09c32514d3b864e5739a222de233c7b47684705/apps/test/util/deprecatedChai.js)) and because folks get confused when they see an assertion like

```js
expect(result).to.eventually.equal(5);
```

...and `eventually` doesn't show up anyplace in the chai docs.